vllm/spec_decode/ngram_worker.py

```diff
@@ -48,7 +48,7 @@ def sampler_output(
         self,
         execute_model_req: ExecuteModelRequest,
         sample_len: int,
-    ) -> Tuple[Optional[List[SamplerOutput]], bool]:
+    ) -> Tuple[Optional[List[Optional[SamplerOutput]]], bool]:
```

Comment:
how did you find this? is it from mypy?

Reply:
In line 119, `None` may be appended to the list.

Reply:
hmm do you know why this wasn't checked from mypy?

Reply:
It's probably because cross-directory checking is disabled, and `SamplerOutput` is defined by the current file.